queries/highlights.scm

```diff
@@ -1,16 +1,20 @@
-[
-    (comment)
-    (note)
-] @comment
+((comment) @comment)
```

Comment:
Last time I checked, `nvim-treesitter` didn't use upstream queries, so if you're a Neovim user, you might want to open a similar PR in `nvim-treesitter`. Feel free to tag me as a reviewer if needed.
Otherwise I'll try not to forget to do it :)

Reply:
I'm not familiar with the term "upstream queries". Could you please elaborate, @cbarette?

Reply:
Now I understand, thanks. How about working through the current PRs, merge what is worthy merging and then I'll create a PR upstream. I'd like to add a few other things to tree-sitter-ledger, but I'm uncertain when be able to make the time.

Reply:
Oh?! And thanks for creating tree-sitter-ledger I find it very help- und useful and it got me to start learning tree-sitter 🙂

Reply:
You're welcome and thanks for all the contributions! You're welcome back anytime :)

Reply:
FYI @cbarrete: I've created [nvim-treesitter/nvim-treesitter#4328](https://github.com/nvim-treesitter/nvim-treesitter/pull/4328) to promote the Ledger specific capture groups to nvim-treesitter.